packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -85,20 +86,21 @@ class UnthemedCodeEditor extends PureComponent<Props> {
 
   handleBeforeMount = (monaco: Monaco) => {
     this.monaco = monaco;
-    const { language, getSuggestions, onBeforeEditorMount } = this.props;
 
-    if (getSuggestions) {
-      this.completionCancel = registerSuggestions(monaco, language, getSuggestions);
-    }
+    const { onBeforeEditorMount } = this.props;
 
     onBeforeEditorMount?.(monaco);
   };
 
   handleOnMount = (editor: MonacoEditorType, monaco: Monaco) => {
-    const { onChange, onEditorDidMount } = this.props;
+    const { getSuggestions, language, onChange, onEditorDidMount } = this.props;
 
+    this.modelId = editor.getModel()?.id;
```

Comment:
This doesn't require a URI? :open_mouth:

Reply:
nope, i think because the model is associated with this editor (as opposed to getting it from the global monaco namespace)